src/components/AssistantTextProcessingForm.vue

```diff
@@ -389,7 +389,9 @@ export default {
 							tt.inputShape.style_input.placeholder = t('assistant', 'Shakespeare or an example of the style')
 							tt.inputShape.source_input.placeholder = t('assistant', 'A description of what you need or some original content')
 						} else if (tt.id === 'core:text2text:translate') {
-							tt.inputShapeDefaults.origin_language = 'detect_language'
+							if (!tt.inputShape.origin_language) {
+								tt.inputShapeDefaults.origin_language = 'detect_language'
```

Comment:
I may not be the right person to review this, I don't get why this works :sweat_smile: